etc/install_package_dependencies.sh

```diff
@@ -1,30 +1,28 @@
-# This script can be used to install all the required package dependencies on a clean system
-
+#This script can be used to install all the required package dependencies on a clean system
+ 
 PROJECT_PATH=$(pwd)
 
 # Install Python-PIP
 sudo apt-get update
-sudo apt-get install -y python-pip
+sudo apt-get install -y python3-pip
+
+# Install virtualenv, create a virtual environment and activate it
+sudo -H pip3 install virtualenv
+virtualenv env
+source env/bin/activate
 
 
 # Install Graph-Tool
 sudo apt-get install -y expat
 sudo apt-get install -y libsparsehash-dev
-sudo apt-get install -y gtk+3
+sudo apt-get install -y gtk 3
```

Comment:
This change in package name from "gtk+3" to "gtk 3" leads to installation failure. If we need gtk-3 to be installed, then "gtk+3" or "libgtk-3-dev" seems to be the packages.